### ISSUE TITLE
Issues with Data Transfer from Withings to Garmin: Potential Restrictions in Code or API Categories?

### ISSUE THREAD

**Issue by CyberschmiedeGmbH** [jaroslawhartman/withings-sync#134]
I have imported blood pressure data into Withings via an interface, which I can normally view within Withings. However, these data are not recognized as data records during the transfer to Garmin and are therefore not transferred. In contrast, manually or original created data records for blood pressure measurement are recognized in Withings. Is it possible that there are restrictions in the code concerning the datasets to be read out? Are there categories or something similar in the Withings API? Thank you

**Comment by longstone** [comment]
Is your account set to the required region?

https://github.com/jaroslawhartman/withings-sync/issues/106#issuecomment-1454395955

**Comment by CyberschmiedeGmbH** [comment]
Yes, the region is set to the USA. I can also upload other blood pressure data by "withings-sync" to Garmin without any problems. The synchronization process works, but not all data from Withings are properly selected/chosen. The blood pressure data that was integrated into Withings via interface are not recognized or selected by "withings-Sync".

**Comment by longstone** [comment]
> category | integer1 for real measures, 2 for user objectives.
https://developer.withings.com/api-reference/#tag/measure/operation/measure-getmeas

Categorie programatically set to 1.

> "category": 1,

https://github.com/jaroslawhartman/withings-sync/blob/master/withings_sync/withings2.py#L246

Could be implemented with a feature flag.
